src/perftest_parameters.c

```diff
@@ -163,6 +167,36 @@ static int get_cache_line_size()
 
 	return size;
 }
+#ifdef HAVE_RO
+/******************************************************************************
+  Check PCIe Relaxed Ordering
+
+  Stolen from https://github.com/pciutils/pciutils/blob/master/example.c
+ ******************************************************************************/
+static bool check_pcie_relaxed_ordering_compliant(void) {
+	struct pci_access *pacc;
+	struct pci_dev *dev;
+	unsigned int c;
```

Comment:
warning: variable 'c' set but not used,

Please, remove

Reply:
Fixed. thanks